apps/mappings/exceptions.py

```
@@ -13,14 +13,14 @@
 )
 
-from apps.mappings.models import ImportLog
+from fyle_integrations_imports.models import ImportLog
 from apps.workspaces.models import SageIntacctCredential
 
 logger = logging.getLogger(__name__)
 logger.level = logging.INFO
 
 
-def handle_import_exceptions(func: callable) -> callable:
+def handle_import_exceptions_v2(func: callable) -> callable:
```

Pylint (3.3.7)

[refactor] Too many statements (58/50)

(R0915)